### ISSUE TITLE
MechWarrior 5: Mercenaries never starts up

### ISSUE THREAD

**Issue by aurismat** [doitsujin/dxvk#2865]
As I have mentioned [here in #1273](https://github.com/doitsujin/dxvk/issues/1273#issuecomment-1221483776), MechWarrior 5 never opens up under Windows with dxvk. Both release version and latest master build produce the same issue - upon startup of the game, dxvk seems to throw an error `err:   D3D11DXGIDevice::QueryResourceResidency: Stub`. Game process then remains doing nothing(not even creating a game window) until it's killed manually by Task Manager.

As per @Blisto91's [suggestion](https://github.com/doitsujin/dxvk/issues/1273#issuecomment-1222177797), removing non-default config options had no effect either.

Is this error a cause of user error(missing dependencies, misconfiguration etc.), or is the game outright not working with dxvk under Windows? If it's the former, any help solving them would be highly appreciated.

I'm unable to produce an apitrace at the moment due to apitrace procedure presented in template causing this:
![image](https://user-images.githubusercontent.com/25552404/186023348-9a309191-6da0-4c96-ad86-e5f573ff5008.png)

Attached log files are produced by [this master build of dxvk](https://github.com/doitsujin/dxvk/actions/runs/2906775906).

### Software information
MechWarrior 5: Mercenaries, no DLCs installed. Enabling/disabling mods makes no difference. Changing graphics settings has no difference as the game never launches, I've attempted to do so anyway.

### System information
- GPU: Gigabyte GTX 1660 OC(GV-N1660OC-6GD)
- Driver: 516.94
- DXVK version: 1.10.3
- OS: Windows 10 21H2 build 19044.1889

### Log files
- d3d9.log: no point uploading - file is empty
- d3d11.log: [MechWarrior-Win64-Shipping_d3d11.log](https://github.com/doitsujin/dxvk/files/9397805/MechWarrior-Win64-Shipping_d3d11.log)
- dxgi.log: [MechWarrior-Win64-Shipping_dxgi.log](https://github.com/doitsujin/dxvk/files/9397810/MechWarrior-Win64-Shipping_dxgi.log)

**Comment by doitsujin** [comment]
https://www.protondb.com/app/784080 seems like some windows specific nonsense?

**Comment by aurismat** [comment]
Huh, seems like it.

Closing this under the assumption of not having enough info to figure out the problem under Windows.

**Comment by Blisto91** [comment]
@aurismat a thing you could try is to make sure you have no game overlays running.
As an example [Geforce Experience Overlay](https://github.com/doitsujin/dxvk/issues/2416#issuecomment-1136465324) can mess with some games.

**Comment by aurismat** [comment]
@Blisto91 unless there's something else I've forgotten to disable, I'm pretty confident I don't have GFE overlay as I've used NVCleanInstall to get my Nvidia drivers. Just for the sake of it I've also disabled Steam overlay with identical results.

Additionally, I've asked a friend with whom I play this in coop to additionally test this in order to eliminate hardware issues - identical results. So I reckon the two identical outcomes add up to be Windows related nonsense as suggested by doitsujin.
